pkg/router/template/types.go

```diff
@@ -236,6 +242,18 @@ type CaptureHTTPHeader struct {
 	MaxLength int
 }
 
+// HTTPHeader specifies an HTTP header that should be set/replace/append
+type HTTPHeader struct {
+	// Name specifies an HTTP header name.
+	Name string
+
+	// Value specifies the header value.
+	Value string
+
+	// Action specifies the action to be performed.
+	Action string
```

Comment:
Better to set this up with a `type ActionType string` 
```suggestion
	Action ActionType
}
// ...
type ActionType string
// ..
const (
        // ...
        ActionTypeDelete ActionType = "Delete"
        // ...
       ActionTypeSet ActionType = "Set"
)
```

Reply:
used the Action Type of the API.